css/properties.json

```diff
@@ -5833,6 +5833,21 @@
     "order": "lengthOrPercentageBeforeKeywords",
     "status": "standard"
   },
+  "text-justify": {
+    "syntax": "auto | distribute | inter-character | inter-word | none",
+    "media": "visual",
+    "inherited": true,
+    "animationType": "discrete",
+    "percentages": "no",
+    "groups": [
+      "CSS Text"
+    ],
+    "initial": "auto",
+    "appliesto": "allElements",
+    "computed": "asSpecified",
+    "order": "uniqueOrder",
+    "status": "standard"
```

Comment:
This is confusing for me.  There is a [W3C Last Call Working Draft] from 10 Oct 2013, and a [Editor's Draft](https://drafts.csswg.org/css-text-3/) from 19 April 2017.  The Editor's draft is the one mentioned on [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/text-justify#Specifications), and changes a few of the items here.  It's unclear if this is trying to follow the 2013 spec, the 2017 draft, or is combining them in some way.